hexrd/ui/image_canvas.py

```diff
@@ -649,6 +649,9 @@ def update_azimuthal_integral_plot(self):
         axis.relim()
         axis.autoscale_view(scalex=False)
 
+    def set_azimuthal_integral_plot_y_scale(self, scale):
+        self.azimuthal_integral_axis.set_yscale(scale)
```

Comment:
I think you should check to see if `self.azimuthal_integral_axis` is `None` first. It will be `None` if the image mode is Cartesian or Raw.